integration-tests/features/steps/common.py

```diff
@@ -1401,6 +1401,42 @@ def check_security_issue_existence(context, cve, package):
                         .format(p=package))
 
 
+def check_job_token_attributes(token):
+    attribs = ["limit", "remaining", "reset"]
+    for attr in attribs:
+        assert attr in token
+        assert int(token[attr]) > 0
```

Comment:
I can imagine the `'remaining'` being `0`.
Can we just `assert isinstance(token[attr], int)` ?